emark/backends.py

```diff
@@ -72,8 +72,9 @@ class ConsoleEmailBackendMixin:
 
     def write_message(self, message):
         msg = message.message()
-        payload_count = len(msg.get_payload())
-        if payload_count > 1:
+        original_payload = msg.get_payload()
+        payload_count = len(original_payload)
+        if isinstance(original_payload, list):
```

Comment:
Please use `msg.is_multipart()` instead and just skip for skip the whole shortening thing for single part emails.